arrows/vxl/pixel_feature_extractor.cxx

```diff
@@ -246,26 +246,48 @@ ::filter( kwiver::vital::image_container_sptr input_image )
 
   std::vector< vil_image_view< pix_t > > filtered_images;
 
-  if( enable_color || enable_gray )
-  {
-    const auto vxl_image =
-      convert_to_typed_vil_image_view< pix_t >( input_image );
+  vil_image_view< double > double_variance;
 
+  if( enable_color )
+  {
     // 3 channels
-    if( enable_color )
+    auto color = convert_to_typed_vil_image_view< pix_t >(
+      input_image );
+    filtered_images.push_back( color );
```

Comment:
(As much because the wrapping was odd...)
```suggestion
    filtered_images.push_back(
      convert_to_typed_vil_image_view< pix_t >( input_image ) );
```